questions.md

```diff
@@ -0,0 +1,5 @@
+# Some questions about pull requests:
+
+*  what happens with multiple branches?
+*  how do you resolve merge conflicts?
```

Comment:
There are a few ways to resolve merge conflicts. You can resolve merge conflicts locally by following the instructions on the command line when you try to merge two branches. There are additional instruction [here](https://help.github.com/en/articles/resolving-a-merge-conflict-using-the-command-line). Or you can do it on GitHub when it says you cannot merge changes into master. There should be a button that appears and it will walk you through the code that needs to be fixed. Instructions for that are [here](https://help.github.com/en/articles/resolving-a-merge-conflict-on-github).